### ISSUE TITLE
KeyError: 'skipped_rules'` in IncludeMissingFileRule

### ISSUE THREAD

**Issue by phihos** [ansible/ansible-lint#1013]
##### Summary

Having an empty `pre_tasks:` in the playbook causes `KeyError: 'skipped_rules'` in `IncludeMissingFileRule`.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible --version
ansible 2.9.6
  config file = None
  configured module search path = ['.../.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = .../ansible-lint/venv/lib/python3.8/site-packages/ansible
  executable location = .../ansible-lint/venv/bin/ansible
  python version = 3.8.2 (default, Jul 16 2020, 14:00:26) [GCC 9.3.0]

ansible-lint --version
ansible-lint 4.3.3
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### OS / ENVIRONMENT

Ubuntu Linux 20.04

##### STEPS TO REPRODUCE

Consider the following playbook:
```yaml
- name: Play
  hosts: all
  pre_tasks:
  tasks:
    - name: Ping
      ping:
```

I tested it with ansible-playbook (Ansible 2.9.6) and it works.

##### Desired Behaviour

ansible-lint should run without any exception.

##### Actual Behaviour

With ansible-lint I get
```
...
File ".../ansible-lint/lib/ansiblelint/rules/IncludeMissingFileRule.py", line 31, in matchplay
  if self.id in task['skipped_rules']:
KeyError: 'skipped_rules'
```

I can not tell why this happens, but it seem to be important that there is a) an empty pre_tasks and b) at least one task. If I remove either the error does not occur.

I have a test case with a fix ready and will open a PR soon.
